### PR TITLE
server: add convenient eval

### PR DESCRIPTION
**Pull request by VitaliyaIoffe**
Syntactic sugar for using net_box eval

### CODE REVIEW

File: luatest/server.lua [hunk 1]
```diff
@@ -208,6 +208,10 @@ function Server:http_request(method, path, options)
     return response
 end
 
+function Server:eval(command)
```
Comment (Steap2448) [inline luatest/server.lua:211]: Arguments and options should be passed as well. See doc [here](https://www.tarantool.io/en/doc/latest/reference/reference_lua/net_box/#lua-function.conn.eval).

File: luatest/server.lua [hunk 2]
```diff
@@ -208,6 +208,10 @@ function Server:http_request(method, path, options)
...
+    return self.net_box:eval(command)
```
Comment (Steap2448) [inline luatest/server.lua:212]:
```suggestion
function Server:eval(command)
    assert(self.net_box, 'net_box is not connected')
    return self.net_box:eval(command)
```

File: luatest/server.lua [hunk 3]
```diff
@@ -208,6 +208,10 @@ function Server:http_request(method, path, options)
...
+end
+
```
Comment (Steap2448) [inline luatest/server.lua:214]: Consider adding a corresponding function for `net_box:call`.

**Review by Steap2448 — CHANGES_REQUESTED** [review]
Thanks for your patch! Consider the following comments.

File: luatest/server.lua (hunk 2 above)
Comment (rosik) [reply luatest/server.lua:212]:
```suggestion
function Server:eval(...)
    assert(self.net_box, 'net_box is not connected')
    return self.net_box:eval(...)

**Comment by rosik** [comment]
I like the idea. Please, make use of the new function:

```console
$ grep -Rn 'net_box:eval'
README.rst:280:    server.net_box:eval('return do_something(...)', {arg1, arg2})
luatest/server.lua:212:    self.net_box:eval('require("luatest.coverage_utils").' .. action .. '()')
test/server_test.lua:111:    t.assert_equals(server.net_box:eval('return os.getenv("custom_env")'), 'test_value')
```

File: luatest/server.lua (hunk 2 above)
Comment (VitaliyaIoffe) [reply luatest/server.lua]: Thank you! Fixed

**Review by Steap2448 — APPROVED** [review]
Nice one!

**Review by rosik — APPROVED** [review]

**Pull request merged**